clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -86,10 +87,18 @@ def get_resource_allocation(self, resource_name: str) -> float:
         )
 
     def get_resource_total(self, resource_name: str) -> float:
-        return sum(
+        # Total node resources minus any allocated resources to excluded pods
```

Comment:
This makes it clear what this code does, is there a reason we want task count and resources to stay in sync like this  with the handling of the excluded resources? If so, I'd imagine this potentially working with the code/config that defines total node resources, like would we benefit from a one-time decrease by 0.6cpu/1000mb (static) or `sum(excluded_resources)` (dynamic) to offset this?

Reply:
Ah you are absolutely right, I missed that we use the total node resources calculation elsewhere.  I agree it makes more sense to move this logic to that code instead.

Reply:
The exclusion was moved to where we calculate total per-node resources, so all code that fetched "total" node resources now report only the total minus excluded resources consistently.